src/haddock/libs/libalign.py

```diff
@@ -61,6 +61,30 @@
 DNA_RES = ["DA", "DC", "DT", "DG"]
```

Comment:
Do you also have RNA defined?

Reply:
> Do you also have RNA defined?

not explicitly, but the function should use by default all the heavy atoms whenever the structure is not dna or protein

Reply:
worth to test it by the way!